packages/help-center/src/components/help-center-header.tsx

```diff
@@ -84,13 +84,14 @@ const HelpCenterHeader: React.FC< Header > = ( {
 							</Route>
 							<Route path="/contact-form" component={ SupportModeTitle }></Route>
 							<Route path="/post" component={ ArticleTitle }></Route>
+							<Route path="/inline-chat">{ __( 'Live Chat', __i18n_text_domain__ ) }</Route>
 						</Switch>
 					) : (
 						__( 'Help Center', __i18n_text_domain__ )
 					) }
-					{ isMinimized && unreadCount ? (
+					{ isMinimized && unreadCount && (
 						<span className="help-center-header__unread-count">{ formattedUnreadCount }</span>
-					) : null }
+					) }
```

Comment:
You'd get a `0` here if `unreadCount` is zero. Because 

`false && 0` returns `0`, which is a valid React child.

Reply:
Adjusted this to be if larger than 0.

The new version is setting the default state at 0 ⇢ https://github.com/Automattic/wp-calypso/pull/67353/files#diff-e31d9874fd791d8bd14201b2906d19b6881e4ab84622c97b4c11f0cb0f29b48dR27